client/src/__snapshots__/PdfGenerator.test.js.snap

```diff
@@ -1,29 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`PdfGenerator renders 1`] = `
-<div>
-  <h1>
-    PDF Generator Tool
-  </h1>
-  <label>
-    PID: 
-    <input
-      id="pdfGeneratePid"
-      onChange={[Function]}
-      type="text"
-      value=""
-    />
-  </label>
-  <button
-    onClick={[Function]}
-  >
-    Generate
-  </button>
-  <h2>
-    Results:
-  </h2>
-  <pre
-    id="pdfGenerateResults"
-  />
-</div>
+<ContextProvider
+  value={
+    Object {
+      "dispatch": [Function],
+      "state": Object {
+        "previousParams": null,
+        "token": null,
+      },
+    }
+  }
+>
+  <PdfGenerator />
```

Comment:
Is it intended that the PdfGenerator snapshot no longer actually includes the rendered content of the PDF Generator? Has the introduction of the ContextProvider had unanticipated side effects on snapshot testing?

Reply:
Made change.